auth/authz_blackbox_test.go

```diff
@@ -339,12 +393,10 @@ func cleanKeycloakResources(t *testing.T) {
 	err = json.Unmarshal([]byte(jsonString), &result)
 	require.Nil(t, err)
 	for _, res := range result {
-		if strings.Contains(strings.ToLower(res.Uri), "test") {
-			deleteResource(t, ctx, res.ID, authzEndpoint, pat)
-		}
+		deleteResource(t, ctx, res.ID, authzEndpoint, pat)
 	}
 
-	policyEndpoint := clientsEndpoint + "/" + clientId + "/authz/resource-server/policy?first=0&max=1000&permission=false"
+	policyEndpoint := clientsEndpoint + "/" + clientId + "/authz/resource-server/policy?first=0&max=1000&name=test&permission=false"
```

Comment:
first=0&max=1000 is just pagination.
name=test is to filter policies which have "test" in the name.
permission=false I guess means we to filter out the permissions and return policies only. But I don't know for sure :) There is no documentation for that API. I just use what they use in the console.